### PR TITLE
Add total queue size metric to CloudWatch

### PR DESCRIPTION
**Pull request by monsha**
It create a new `Summary` dimension with Total Queue Size metric.

```
{
  namespace: "Huginn/Sidekiq",
  metric_data: {
    metric_name: "Size",
    dimensions: [
      {
        name: "Summary",
        value: "Total"
      }
    ],
    value: 9999,
    unit: "Count",
    timestamp: timestamp
}
```

### CODE REVIEW

File: lib/tasks/sidekiq_metrics.rake [hunk 1]
```diff
@@ -10,9 +10,12 @@ namespace :sidekiq do
         latency: { value: queue.latency.to_i, unit: 'Seconds' }
       }
     end
+    total_size = queues.values.reduce(0) { |sum, e| sum + e[:size][:value] }
+    summary = Hash["Total", { size: { value: total_size, unit: "Count" } }]
```
Comment (OpakAlex) [inline lib/tasks/sidekiq_metrics.rake]: cool

**Review by OpakAlex — APPROVED** [review]

**Comment by duboff** [comment]
Nice, can we also add `Max Latency` metric which will measure largest latency across queues?

**Comment by monsha** [comment]
@duboff done!

**Comment by duboff** [comment]
👍

**Pull request merged**